### PR TITLE
Update juliamono w/ git auto-update

### PR DESCRIPTION
**Pull request by cormullion**
Adding CSS files.

### CODE REVIEW

File: packages/j/juliamono.json [hunk 1]
```diff
@@ -14,14 +14,16 @@
   },
   "license": "OFL-1.1",
   "homepage": "https://github.com/cormullion/juliamono",
+  "filename": "webfonts/juliamono-regular.css",
```
Comment (mortenpi) [inline packages/j/juliamono.json:17]: Should it maybe be this, because `basePath = "webfonts"`?

```suggestion
  "filename": "juliamono-regular.css",
```

Comment (cormullion) [reply packages/j/juliamono.json]: That might be a better guess than mine! :)

**Comment by cormullion** [comment]
Thanks!

**Comment by cormullion** [comment]
@MattIPv4 Do I have to do something else to get this package to auto-update?

**Comment by MattIPv4** [comment]
👋 We have ongoing maintenance with the auto-update functionality of cdnjs currently -- this change was made since the maintenance started, and so our auto-update logic hasn't been running to detect it. You can follow along on our status page for when the maintenance will conclude: https://status.cdnjs.com/

**Comment by cormullion** [comment]
Ok, thanks!

**Pull request merged**